subprojects/scala/src/integTest/groovy/org/gradle/scala/compile/CachedScalaCompileIntegrationTest.groovy

```diff
@@ -40,6 +40,15 @@ class CachedScalaCompileIntegrationTest extends AbstractCachedCompileIntegration
             dependencies {
                 compile group: 'org.scala-lang', name: 'scala-library', version: '2.11.8'
             }
+            
```

Comment:
Is this some leftover debug code?

Reply:
Thanks, removed. Please add comments below the marked code as only code above the comment shows up on the conversation page.